app/controllers/questions_controller.rb

```diff
@@ -37,6 +37,10 @@ def create
     redirect_to new_question_path, notice: msg
   end
 
+  def edit
+    flash[:from] = params[:from] if params.has_key? :from
+  end
+
```

Comment:
What does this function do? Flash is traditionally used for messages or alerts. Are you using it in another way here?